hangman.py

```diff
@@ -239,7 +256,7 @@ def user_wants_hints():
 
     while True:
```

Comment:
It is not good to use True in condition of while loop. Change it to boolean var.